ontotext-yasgui-web-component/src/services/yasgui/yasgui-builder.ts

```diff
@@ -17,7 +25,7 @@ export class YasguiBuilder {
   build(hostElement: HTMLElement, yasguiConfiguration: YasguiConfiguration): OntotextYasgui {
     // @ts-ignore
     const yasgui = new Yasgui(HtmlElementsUtil.getOntotextYasgui(hostElement), yasguiConfiguration.yasguiConfig);
-
+    this.yasrService.setYasgui(yasgui);
```

Comment:
Can this be set in the serviceFactory too? I guess it can't because the build happens later, but now this service initialization happens on two steps on two different places and is quite ugly.